api/repositories/process_repository.go

```diff
@@ -102,7 +110,27 @@ func (r *ProcessRepo) GetProcess(ctx context.Context, authInfo authorization.Inf
 	if err != nil { // untested
 		return ProcessRecord{}, err
 	}
-	return returnProcess(processList.Items)
```

Comment:
I would suggest renaming `returnProcess` to `ensureSingleProcess` or something similar. `returnProcess` sounds like a `return` statement to me and I would expect no code after it.